### DISCUSSION TITLE
better-sqlite3 won't install

### DISCUSSION THREAD

**Discussion by lyndondrake** [aidenlx/obsidian-zotero#320] [General]
I've installed the ZotLit plugin from the Community Plugins repository. When I try to enable it, I get the message about installing sqlite. The auto install doesn't work, and when I download it and select the downloaded file, there is no additional message but when I restart the plugin I get the same message. Any ideas on how to get it working?

**Comment by matar3** [comment]
The same for me. I'm on Windows, and have already installed latest version of node.js (v20.12.0 ) together with node-gyp (v. 10.1.0). I tried to follow troubleshooting instructions for better-sqlite3 ( https://github.com/WiseLibs/better-sqlite3/blob/master/docs/troubleshooting.md), but i'm not understanding if i have to use MS visualstudio, do a global installation for better-sqlite3, or not; i tried to install better-sqlite3 globally (using the command "npm install better-sqlite3 -g), but in obsidian the plugin still can't enable saying that better-sqlite3 is missing or broken. Please Help!

**Comment by aidenlx** [comment]
v2 uses a new database layer with no native binary, so better-sqlite3 is out of the picture entirely — nothing to compile, nothing to download, and no node-gyp or Visual Studio toolchain involved. Point the plugin at your Zotero data directory and it reads the database directly.

Reading while Zotero is open is more reliable now too.

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
